src/modules/admins/models/scam-update.input.ts

```diff
@@ -0,0 +1,24 @@
+import { InputType, Field, registerEnumType } from '@nestjs/graphql';
+import { Matches } from 'class-validator';
+import { COLLECTION_IDENTIFIER_ERROR, COLLECTION_IDENTIFIER_RGX } from 'src/utils/constants';
+
+@InputType()
+export class ScamUpdateInput {
+  @Matches(RegExp(COLLECTION_IDENTIFIER_RGX), {
+    message: COLLECTION_IDENTIFIER_ERROR,
+  })
+  @Field(() => String)
+  collectionIdentifier: string;
+
+  @Field(() => ScamInputEnum)
+  type: ScamInputEnum;
+}
+
+export enum ScamInputEnum {
+  deny = 'deny',
```

Comment:
Maybe use PascalCase for enums (if applicable)?

Reply:
I need to keep them like this here because of mongo